core/ir/opnd_api.h

```diff
@@ -1770,7 +1775,10 @@ struct _opnd_t {
          * above, to save space.
          */
         instr_t *instr; /* INSTR_kind, FAR_INSTR_kind, and MEM_INSTR_kind */
-        reg_id_t reg;   /* REG_kind */
+        struct {
+            reg_id_t reg;
+            opnd_size_t element_size;
```

Comment:
Add a non-doxygen comment `XXX #5638: Fill in the element size for x86 and aarch32.`?

Reply:
Done